Add a note about amd64 (#3058)
microsoft/AzureTRE: docs/tre-admins/setup-instructions/prerequisites.md

!!! tip
    An alternative of running the development container locally is to use [GitHub Codespaces](https://docs.github.com/en/codespaces).

!!! warning

>     This solution and procedures described throughout the documentation have been tested on machines using ***amd64 architecture*** CPUs. We have limited access to other types of machines and some issues might exist if those are used.

Reviewer: ```suggestion
    The procedures described throughout the documentation have been tested using machines with ***amd64 architecture*** CPUs. We have limited access to machines with other CPU architectures and issues may be encountered if those are used.
```
